content/info/GDPR.md

```diff
@@ -10,7 +11,7 @@ This website is hosted on [GitHub](https://github.com/), the hoster might be col
 
 We keep your emails on [ProtonMail](https://protonmail.com/) servers in accordance with [their privacy policy](https://protonmail.com/privacy-policy). We are not sharing them with anyone and will ask for your consent to add your address to any mailing lists in a separate email.
 
-If you would like us to delete the personal data we have or make any changes, please [send us an email](mailto:contact@zymologia.fi).
+If you would like us to delete the personal data we have or make any changes, please [send us an email](@/info/contacts.md).
```

Comment:
This operation should have minimum number of clicks involved; maybe just `mailto` makes more sense here